index.html

```diff
@@ -63,28 +63,50 @@ <h2 id="project_tagline"></h2>
 <pre>$ java -jar publish.war -h
 You passed help flag.
 usage: java -jar
- -en,--exchange_name &lt;arg&gt;    exchange name, default is amq.direct
- -f,--content_file &lt;arg&gt;      event content file
- -h,--help                    show help.
- -json,--json_content &lt;arg&gt;   event content in json string. The value can
-                              also be a dash(-) and the json will be read
-                              from the output of other programs if piped.
- -mb,--message_bus &lt;arg&gt;      host of message bus to use, default is
-                              127.0.0.1
- -np,--non_persistent         remove persistence from message sending
- -port,--port &lt;arg&gt;           port to connect to message bus
- -rk,--routing_key &lt;arg&gt;      routing key, mandatory</pre>
+ -d,--debug                     enable debug traces
+ -domain,--domain &lt;arg&gt;         domain
```

Comment:
I think we should use the term domanId in this document. Some explanation should be used. Perhaps we can steal a line or two from here: https://github.com/Ericsson/eiffel/blob/master/eiffel-syntax-and-usage/the-meta-object.md#metasourcedomainid
Perhaps we should add a link to that as well?

Perhaps we should change -domain, --domain to -domain, --domainId?